package.json

```diff
@@ -1,18 +1,31 @@
 {
   "name": "llnode",
-  "version": "1.4.0",
+  "version": "0.0.0",
   "description": "llnode test suite",
   "main": "no-entry-sorry.js",
   "directories": {
     "test": "test"
   },
+  "//": "We need to use the install scripts as llnode must be built on the machine it's deployed on.",
+  "//": "(Blame C++)",
   "scripts": {
+    "preinstall" : "node scripts/configure.js",
+    "install" : "./gyp_llnode && make -C out/",
```

Comment:
I think you'd have to ask @indutny why he originally didn't use node-gyp. I would add though that adding that to the build process adds a dependency on having node. When you build llnode from github there's not actually any dependancy on node.

There's a major dependancy on having the right lldb headers and you do need gyp in some form but weirdly for a node debugging tool there's no dependancy on node itself. llnode is built against a specific version of lldb but will work against a wide range of node versions - including being able to open Linux cores with lldb/llnode on Mac. If we did move to node-gyp we'd want to avoid introducing a link to the version of node that was running node-gyp.

For my part I wanted to focus on making the changes necessary to configure and build via npm to make it easier for people to get hold of rather than re-writing the build process. Moving to node-gyp isn't necessarily a bad idea but it should probably be raised as a separate issue.